src/lecture_event/lectureEvent.types.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose, { Document } from 'mongoose';
+
+export interface ILectureEvent extends Document {
+  _id: mongoose.Types.ObjectId;
+  startDateTime: Date;
+  endDateTime: Date;
+  title: string;
+  groups: number[];
+  subgroups?: [number];
+  lecturer: string;
+  lectureTypes: [string];
```

Comment:
when you are working with typescript and you want to specify an array of some type use:
type[], so in this case string[]